spec/features/users/authentication_spec.rb

```diff
@@ -11,4 +15,11 @@
     log_in user
     expect(page).to have_content('Vesta')
   end
+  it 'allows users to register' do
+    visit '/users/sign_up'
+    default_sign_up.each { |a, v| fill_in a.to_s, with: v }
+    select('male', from: 'user_gender')
```

Comment:
I think that this can be extracted to a `fill_in_registration` helper method that includes the attributes hash, then this is ready to merge!